File: nodejs/http-server.js
const express = require('express')

const app = express()

const port = 2026

app.use(express.json())

app.post('/', (req, res) => res.send('ok'))

app.post('/req', (req, res) => {
    console.log(`Request => ${JSON.stringify(req.body)}`)
    res.send('ok')
})

app.post('/res', (req, res) => {
    console.log(`Response => ${JSON.stringify(req.body)}`)
    res.send('ok')
})

app.listen(port, () => console.log(`js-hook-server ${port}!`))

File: python/http_server.py
from fastapi import FastAPI, Request

app = FastAPI()

@app.get("/")
async def read_root():
    return "ok"

@app.post("/req")
async def handle_request(request: Request):
    body = await request.json()
    print(f"Request => {body}")
    return "ok"

@app.post("/res")
async def handle_response(request: Request):
    body = await request.json()
    print(f"Response => {body}")
    return "ok"


# run: uvicorn main:app --reload --port 3000
